src/_info.sh

```diff
@@ -26,13 +26,14 @@ function mod_info() {
   msg "    %s\n" "$active"
   echo >&2 ""
 
+  #Check for Maintenance Mode
+  msg `check_for_maintenance_mode`
```

Comment:
I missed this before but - I don't think you need to wrap this in the `msg` plus backticks, as the function itself is already calling `info`.